FlowCryptUI/Nodes/PassPhraseAlertNode.swift

```diff
@@ -13,6 +13,9 @@ public class PassPhraseAlertNode: ASDisplayNode {
     enum Constants {
         static let antiBruteForceProtectionAttemptsMaxValue = 5
         static let blockingTimeInSeconds: Double = 5 * 60
+        static let textColor = UIColor.colorFor(darkStyle: .white, lightStyle: .black)
+        static let okayButtonColor = UIColor(hex: "4591FC") ?? .blue
+        static let buttonFont = UIFont.boldSystemFont(ofSize: 16)
```

Comment:
why did you decide to go with bold font? previous regular looked more similar to system alert

Reply:
Because this one looked like bold
![image](https://github.com/FlowCrypt/flowcrypt-ios/assets/77344191/ae9e557c-21ac-4a55-a037-0e9107081283)

Reply:
Do you wanna me to go with regular font?

Reply:
ah, here `Cancel` is bold and `Set` is regular, let's use regular for passphrase alert

Reply:
And for colors it'll be better to use `UIColor.systemRed` and `UIColor.systemBlue`, they're the same as colors of system alerts